auth-jwt-service/src/test/java/io/stargate/auth/jwt/AuthnJwtServiceTest.java

```diff
@@ -28,7 +28,6 @@
 import com.nimbusds.jose.proc.SecurityContext;
 import com.nimbusds.jose.proc.SimpleSecurityContext;
 import com.nimbusds.jwt.JWTClaimsSet;
-import com.nimbusds.jwt.JWTClaimsSet.Builder;
```

Comment:
It's just a warning, the rationale is that importing common names like "Builder" can make the code harder to read:

> **BadImport**
> Importing nested classes/static methods/static fields with commonly-used names can make code harder to read, because it may not be clear from the context exactly which type is being referred to. Qualifying the name with that of the containing class can make the code clearer.

I tend to agree. In this class it's still pretty clear because the builder is used close to its parent class:
```java
JWTClaimsSet jwtClaimsSet = new Builder().claim("stargate_claims", stargate_claims).build();
```
But in the general case it can become unclear, for example if we store the builder in a field or pass it as a method parameter.